### PR TITLE
deleted some header files

### PR DESCRIPTION
**Pull request by intOppong**
improvements

### CODE REVIEW

File: file.c [hunk 1]
```diff
@@ -1,12 +1,7 @@
-
+// Money Trees
```
Comment (maxOppong) [inline file.c]: what does this mean

Comment (intOppong) [inline file.c]: nothing

File: file.c [hunk 2]
```diff
@@ -1,12 +1,7 @@
-
+// Money Trees
 
 #include <stdio.h>
-#include <stdlib.h>
```
Comment (maxOppong) [inline file.c]: we need this header file

File: file.c [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // Money Trees
 
 #include <stdio.h>
+#include <stdlib.h>
```
Comment (maxOppong) [inline file.c]: cool

**Pull request merged**